src/api/spec/models/staging/workflow_spec.rb

```diff
@@ -23,22 +23,19 @@
     staging_workflow
   end
 
+  context 'when created' do
+    let(:role) { Role.find_by_title('reviewer') }
+    it { expect(project.relationships.where(group: group, role: role).size).to eq(1) }
```

Comment:
[nitpick] the flag tests use the form `expect(project.relationships.where(group: group, role: role)).to exist`, which is a little easier.

Reply:
done!